<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>EEPROM</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_AdresseEE">
           <property name="text">
            <string>adresseEE :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="del_adresseEE">
           <property name="text">
            <string>del_adresseEE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_EE1">
              <property name="text">
               <string>EE1 :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_EE2">
              <property name="text">
               <string>EE2 :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_EE3">
              <property name="text">
               <string>EE3 :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_EE4">
              <property name="text">
               <string>EE4 :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_EE5">
              <property name="text">
               <string>EE5 :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="del_EE1">
              <property name="text">
               <string>del_EE1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="del_EE2">
              <property name="text">
               <string>del_EE2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="del_EE3">
              <property name="text">
               <string>del_EE3</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="del_EE4">
              <property name="text">
               <string>del_EE4</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="del_EE5">
              <property name="text">
               <string>del_EE5</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Bus</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_busAdresse">
              <property name="text">
               <string>busAdresse :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_busDonnee">
              <property name="text">
               <string>busDonnee :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="del_busAdresse">
              <property name="text">
               <string>del_busAdresse</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="del_busDonnee">
              <property name="text">
               <string>del_busDonnee</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Instruction</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_Inst">
              <property name="text">
               <string>Inst :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="del_Inst">
              <property name="text">
               <string>del_Inst</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ASM">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QGroupBox" name="groupBox_10">
              <property name="title">
               <string>Incr√©menteur 16 bits</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_inc16">
                 <property name="text">
                  <string>inc16 :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="del_inc16">
                 <property name="text">
                  <string>del_inc16</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_Inc">
                 <property name="text">
                  <string>Inc :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="del_Inc">
                 <property name="text">
                  <string>del_Inc</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_11">
              <property name="title">
               <string>Program Counter</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_PC">
                 <property name="text">
                  <string>PC :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="del_PC">
                 <property name="text">
                  <string>del_PC</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_10" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Registre XY</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_XY">
                 <property name="text">
                  <string>XY :</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_X">
                 <property name="text">
                  <string>X :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="del_X">
                 <property name="text">
                  <string>del_X</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_Y">
                 <property name="text">
                  <string>Y :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="del_Y">
                 <property name="text">
                  <string>del_Y</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Registre M</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_M">
                 <property name="text">
                  <string>M :</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_M1">
                 <property name="text">
                  <string>M1 :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="del_M1">
                 <property name="text">
                  <string>del_M1</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_M2">
                 <property name="text">
                  <string>M2 :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="del_M2">
                 <property name="text">
                  <string>del_M2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>Registre J</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_J">
                 <property name="text">
                  <string>J :</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_J1">
                 <property name="text">
                  <string>J1 :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="del_J1">
                 <property name="text">
                  <string>del_J1</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="label_J2">
                 <property name="text">
                  <string>J2 :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="del_J2">
                 <property name="text">
                  <string>del_J2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Registres</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_A">
           <property name="text">
            <string>A :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="del_A">
           <property name="text">
            <string>del_A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_B">
           <property name="text">
            <string>B :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="del_B">
           <property name="text">
            <string>del_B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_C">
           <property name="text">
            <string>C :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="del_C">
           <property name="text">
            <string>del_C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_D">
           <property name="text">
            <string>D :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="del_D">
           <property name="text">
            <string>del_D</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>ALU</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="3">
             <widget class="QLabel" name="label_Cy">
              <property name="text">
               <string>Cy :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="del_Cy">
              <property name="text">
               <string>del_Cy</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_Z">
              <property name="text">
               <string>Z :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="del_Z">
              <property name="text">
               <string>del_Z</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_S">
              <property name="text">
               <string>S :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="del_S">
              <property name="text">
               <string>del_S</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_codeALU">
              <property name="text">
               <string>codeALU :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="del_codeALU">
              <property name="text">
               <string>del_codeALU</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ALU">
           <property name="text">
            <string>ALU :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="del_ALU">
           <property name="text">
            <string>del_ALU</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>Contr√¥le</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="0">
             <widget class="QLabel" name="del_Clock">
              <property name="text">
               <string>del_Clock</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_Clock">
              <property name="text">
               <string>Clock :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_nbrCycles">
              <property name="text">
               <string>nbrCycles:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_Run">
              <property name="text">
               <string>Run</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Inst">
              <property name="text">
               <string>Inst</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Step">
              <property name="text">
               <string>Step</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_modeFonctionnement">
           <property name="text">
            <string>Mode de Fonctionnement : STEP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_11" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Fr√©quence (Hz)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_timer">
           <property name="text">
            <string>Periode : 200ms </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
